src/Components/Tooltip.tsx

```diff
@@ -1,16 +1,19 @@
 import React from "react"
 import styled, { StyledFunction } from "styled-components"
 import colors from "../Assets/Colors"
+import { unica } from "Assets/Fonts"
 
 interface Props extends React.HTMLProps<HTMLDivElement> {
-  message: string
-  hoverAlign?: string
+  hAlign?: string
   hoverWidth?: number
+  message: string
+  vAlign?: string
```

Comment:
Is this ‘horizontal’ and ‘vertical’? If so, I’d prefer to call them that and have the options be entirely self-explanatory.

Reply:
yea. I thought it's a pretty common abbreviation. But you are right - explicit is better. Updated